bemani/api/objects/records.py

```diff
@@ -89,6 +89,7 @@ def __format_jubeat_record(self, record: Score) -> Dict[str, Any]:
             'status': status,
             'combo': record.data.get_int('combo', -1),
             'ghost': ghost,
+            'music_rate': record.data.get_int('music_rate', -1)
```

Comment:
This will need a corresponding update to the API specification. What's music_rate used for?

Reply:
music_rate is a percentage based on the number of goods, greats, and perfects you get in a song. It was added in festo and is required for its jubility calculation. It's stored as an int internally though and goes up to 120% because of hard mode

Reply:
Ah, good to know. The spec will need updating as well then (https://github.com/DragonMinded/bemapi)